libraries/AC_Avoidance/AC_Avoid_config.h

```diff
@@ -0,0 +1,12 @@
+
+#pragma once
+
+#include <AP_HAL/AP_HAL_Boards.h>
+
+#ifndef AC_AVOID_ENABLED
+#define AC_AVOID_ENABLED 1
+#endif
+
+#ifndef AP_OAPATHPLANNER_ENABLED
+#define AP_OAPATHPLANNER_ENABLED AC_AVOID_ENABLED
+#endif
```

Comment:
```suggestion
#endif

#ifndef AP_OAPATHPLANNER_BACKEND_DEFAULT_ENABLED
#define AP_OAPATHPLANNER_BACKEND_DEFAULT_ENABLED AP_OAPATHPLANNER_ENABLED
#endif

#ifndef AP_OAPATHPLANNER_BENDYRULER_ENABLED
#define AP_OATHPATHPLANNER_BENDYRULER_ENABLED  AP_OAPATHPLANNER_BACKEND_DEFAULT_ENABLED
#endif


```

... and dijkstra too

Reply:
Done. Thanks.